automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -108,6 +108,7 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
                   workspaceListPage.validateLocation()
                   // TODO: add assertions for the new "request access" modal
                   // TODO: end test somewhere we can access the sign out button
+                  new RequestAccessModal().cancel()
```

Comment:
This seems brittle. If the modal doesn't appear, then this will fail. I think that could happen if we change the way the FiaB is populated, e.g. who's the owner on each billing project. It would probably be better to just do a `page.closeModal()` here (and comment that it's the request access modal).